src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Altinn.Platform.Authorization.csproj

```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
 
  <ItemGroup>
-    <PackageReference Include="AltinnCore.ServiceLibrary" Version="9.0.1" />
-    <PackageReference Include="JWTCookieAuthentication" Version="0.8.9-alpha" />
+    <PackageReference Include="AltinnCore.ServiceLibrary" Version="9.0.2" />
```

Comment:
Should we update the AltinnCore projects aswell with the nuget package?

Reply:
I have updated all components that had a reference to the nuget package